inventory/serializers.py

```diff
@@ -90,7 +90,40 @@ def create(self, validated_data):
         return order
 
 
-class LoanOrderSerializer(serializers.ModelSerializer):
+class LoanOrderSerializer(OrderSerializer):
+    order_items = OrderItemSerializer(many=True)
+
     class Meta:
         model = LoanOrder
-        fields = "__all__"
+        fields = [
+            "action",
+            "reason",
+            "date",
+            "user",
+            "other_info",
+            "order_items",
+            "loanee_name",
+            "return_date",
+            "loan_active",
+        ]
+
+    def create(self, validated_data):
+        order_items_data = validated_data.pop("order_items")
+
+        validated_data["user"] = self.context["request"].user
+        order = LoanOrder.objects.create(**validated_data)
+
+        for order_item_data in order_items_data:
+            OrderItem.objects.create(order=order, **order_item_data)
+
+        return order
+
+
+class ActionTypeSerializer(serializers.Serializer):
+    CHOICES = (
+        ("Withdraw", "Withdraw"),
+        ("Deposit", "Deposit"),
+    )
+
+    action = serializers.ChoiceField(choices=CHOICES)
```

Comment:
we have a global choices variable for withdraw and deposit `action_choices` that can be imported from `.globals` 

```python
from .globals import action_choices
action = serializers.ChoiceField(choices=action_choices)
```